images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -284,6 +284,60 @@ meteringconfig_storage_s3Compatible_create_secret: "{{ _storage_spec | json_quer
 meteringconfig_storage_s3Compatible_credentials_secret_name: "{{ _storage_spec | json_query('hive.s3Compatible.secretName') }}"
 meteringconfig_storage_s3Compatible_endpoint: "{{ _storage_spec | json_query('hive.s3Compatible.endpoint') }}"
 
+#
+# Override the default values to handle Openshift, and non-Openshift configurations
+#
+meteringconfig_ocp_disabled: "{{ lookup('env', 'DISABLE_OCP_FEATURES') | bool or meteringconfig_default_values | combine(meteringconfig_spec_overrides, recursive=True) | json_query('disableOCPFeatures.enabled') }}"
+_meteringconfig_ocp_disabled_overrides: "{{ meteringconfig_ocp_disabled | ternary(_ocp_disabled_overrides, _ocp_enabled_overrides) }}"
```

Comment:
If `meteringconfig_ocp_disabled` is false, then instead of assigning these overrides w/ an empty dictionary, it assigns them the `_ocp_enabled_overrides`, which only handles setting the prometheus url.

Reply:
Awesome.